client/src/ui.rs

```diff
@@ -30,15 +37,8 @@ fn app() -> Html {
     html! {
         <>
```

Comment:
Empty tag?

Reply:
Its the start of a special html fragrant tag. A way to return more that one element from a view. That is normally not allowed .